widgets/AttributesTable/_GraphicsMixin.js

```diff
@@ -648,15 +648,45 @@ define([
             this.clearGraphicsLayer(this.bufferGraphics);
         },
 
-        clearGraphicsLayer: function (layer) {
+        clearGraphicsLayer: function (layer, specificFeatures) {
```

Comment:
the argument should be `specificGraphics` so it can be generalized for any graphic layer.